src/redux/books/books.js

```diff
@@ -1,34 +1,69 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
 const ADD_BOOK = 'bookstore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+const GET_BOOK = 'bookstore/books/GET_BOOK';
 
 const prevState = [];
-
-// Action Logic
-const addBook = (book) => ({
-  type: ADD_BOOK,
-  book,
-});
-
-const removeBook = (id) => ({
-  type: REMOVE_BOOK,
-  id,
-});
+const source = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/BoLpCLs90yg1VqJcibSz/books';
 
 // Book Reducer logic
 const BooksRedux = (state = prevState, action) => {
   switch (action.type) {
+    case GET_BOOK:
+      return action.books;
+
     case ADD_BOOK:
       return [
         ...state,
         action.book,
       ];
     case REMOVE_BOOK:
-      return [
-        ...state.filter((book) => book.id !== action.id),
-      ];
+      return state.filter((book) => book.item_id !== action.itemB_id);
```

Comment:
- Currently, when I delete a book from your app, the app doesn't get updated until I refresh the page. A likely reason for this could be that you are accessing the `id` from the action creator wrongly. I suggest you use `action.itemB.id`. Please fix this 🙏 🙏